eas.json

```diff
@@ -16,14 +16,20 @@
       "ios": {
         "simulator": true,
         "resourceClass": "large"
+      },
+      "env": {
+        "EXPO_PUBLIC_ENV": "production"
       }
     },
     "preview": {
       "extends": "base",
       "distribution": "internal",
-      "channel": "preview",
+      "channel": "production",
       "ios": {
         "resourceClass": "large"
+      },
+      "env": {
+        "EXPO_PUBLIC_ENV": "production"
       }
     },
```

Comment:
I think this `preview` profile was only added for APK builds (happens bc of `distribution: 'internal'`). Could maybe remove if we can build APKs with the `testflight` profile.

Reply:
Ah interesting. I think what I'll do for now then is set this one to use `testflight` channel, since we'll still need to do Android builds locally until we figure out the Google Play alpha builds stuff.